### ISSUE TITLE
Issues with Total Variation loss

### ISSUE THREAD

**Issue by mr-johnlocke8** [sergeywong/cp-vton#4]
Hey, I get these results after I've added Total Variation loss (https://discuss.pytorch.org/t/yet-another-post-on-custom-loss-functions/14552).

<img width="1069" alt="screen shot 2018-12-16 at 20 14 51" src="https://user-images.githubusercontent.com/44749945/50057181-6e988e00-016f-11e9-9fb7-d42c9e41908b.png">

What can cause this problem?

**Comment by solitarysandman** [comment]
Are you sure you've implemented TV loss correctly?

**Comment by tuan-seoultech** [comment]
![image](https://user-images.githubusercontent.com/31918737/58217923-dfa42900-7d3f-11e9-88f5-a2cd19ae98b2.png)
What is the difference between A and B?
How do you add TV loss? So, Total loss = L1 loss + TV loss?
Anyway, I think TV loss is not suitable for this because TV loss doesn't care the shape and position of pixels inside cloth image.
